config/log.conf

```diff
@@ -1,4 +1,4 @@
-log_format main '$remote_addr - $remote_user [$time_local]  $status '
+log_format main '$remote_addr - $remote_user [$time_local] $status '
```

Comment:
Removed because I think this was left here by mistake. Not easy to spot so it could have easily been missed.